pacstall

```diff
@@ -207,7 +207,7 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
       if [[ ! -e "$STGDIR/repo/" ]]; then
         sudo mkdir -p "$STGDIR/repo"
         sudo touch "$STGDIR/repo/pacstallrepo.txt"
-	      sudo pacstall -C
+	  sudo pacstall -C
```

Comment:
nope

Reply:
It needed one more space

Reply:
![img](https://0x0.st/-91w.png)
Its fixed...